src/___tests__/statuscake.test.js

```diff
@@ -45,6 +49,46 @@ test('fetchStatuCakeData call axios', async () => {
   expect(storage.setData).toBeCalledTimes(1);
 });
 
+test('it should filter metrics base on config', async () => {
+  axios.get.mockResolvedValue({
+    data: {
```

Comment:
it looks like your data is duplicated, you can put it elswhere as a constant, or even make a util function to build it in different manners

Reply:
Exact, done 🐶